tests/performance/locust_settings.py

```diff
@@ -150,8 +151,19 @@ def _serialize_host(self, url: AnyHttpUrl, info: SerializationInfo) -> str:
 
 
 if __name__ == "__main__":
-    settings = LocustSettings()
-    env_vars = [
-        f"{key}={val}" for key, val in json.loads(settings.model_dump_json()).items()
-    ]
+    locust_settings = LocustSettings()
+
+    arguments = dict(
+        arg.removeprefix("--").split("=") for arg in sys.argv if arg.startswith("--")
+    )
+
+    settings_objects = [locust_settings]
+    for sclass in _get_settings_classes(locust_settings.LOCUST_LOCUSTFILE):
```

Comment:
so you load settings from other modules in `locust_file` and print them all key-value so that you can build a dot-env file

Reply:
yep, exactly